components/admin_console/data_grid/data_grid.tsx

```diff
@@ -61,6 +63,10 @@ type Props = {
         keys: string[];
         onFilter: (options: FilterOptions) => void;
     };
+
+    customGridClass?: string;
```

Comment:
Thanks to @hahmadia for bringing this to my attention, I missed this in my first pass.

We can do this better by just providing one "custom class" to the component here, and letting CSS manage the styles based on the default class names already included.

So instead of having styles written like:
```
.MyCustomGridClass {
    display: block;
}

.MyCustomHeaderClass {
    padding: 20px;
    // some other styles
}
```

We can do this:
```
.MyCustomGridClass {
    display: block;
   
    .DataGrid_header {
        padding: 20px;
        // some other styles
    }
}
```